src/pages/persona/PersonaAnalysis.tsx

```diff
@@ -14,9 +14,9 @@ enum TEXT {
 
 export default function PersonaAnalysisPage(): ReactElement {
   const { data } = useFetch<Viewer>('/persona/count');
-  const countWithComma = data ? data.count.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',') : 0;
+  const countWithComma = data ? data.count.toLocaleString() : 0;
```

Comment:
P4;

요기 `countWithComma`의 경우 string | number가 들어가게 되는데, string으로 통일시켜주면 어떨까요?
```suggestion
  const countWithComma = data ? data.count.toLocaleString() : '0';
```